.bingo/Variables.mk

```diff
@@ -0,0 +1,36 @@
+# Auto generated binary variables helper managed by https://github.com/bwplotka/bingo v0.2.3. DO NOT EDIT.
```

Comment:
Bingo is great, you can use these Makefile variables in any other Makefile, and it will assure that the correct version of the tool is installed and used. Check out `Makefile` to see it in action.